openid-4-verifiable-presentations-1_0.md

```diff
@@ -320,6 +320,24 @@ The following is a non-normative example of a transaction data content, after ba
 }
 ```
 
+`attachments`:
+: OPTIONAL. Array of objects, where each object includes an extra resource that needs to be passed to the Wallet. These resources are intended to provide additional context to the Wallet or the End-User. The Wallet MUST ignore any unrecognized attachment formats. Each object contains the following parameters:
+
+    * `format`: REQUIRED: String that identifies the format of the attachment. The specific values are out of scope of this specification. It is RECOMMENDED to use collision-resistant names for type values.
```

Comment:
I don't have an issue with this being "format", but is there a specific reason transaction data uses "type" and this uses "format"? I think using "type" could make sense so that attachments/transaction data work somewhat consistently (also with "credential_ids"). Then "format" is sort of reserved for credential formats. 

I also think type makes sense as the format here is kinda like a (media) type.

But again, not too important

Reply:
I think the big difference is that this is on a different meta level. An attachment will have both, a format (credential format) and a type (expressed in terms determined by the format). It's basically like in DCQL queries. Transaction data types, in contrast, are always JSON, so we can omit the format.